cargo-apk/README.md

```diff
@@ -159,6 +159,11 @@ orientation = "landscape"
 # Unset by default, or "true" when targeting Android >= 31 (S and up).
 exported = "true"
 
+# See https://developer.android.com/guide/topics/manifest/activity-element#resizeableActivity
+#
+# Unset by default, or "true" when targeting Android >= 24 (Nougat and up).
+resizeable_activity = "true"
```

Comment:
Wondering if this - and `exported = "true"` - should be the TOML literal `true` instead of the string `"true"`, just like `required = true` under `[[package.metadata.android.uses_feature]]`.

Reply:
Probably yes. I just used the other boolean attribute, `exported`, as a template. Shall I change both?

Reply:
Fixed in c388b497bc146e8d625b20f30118d266a1a82007. I also tested that the non-string values work fine as documented in the README.

Reply:
Perfect, thanks a lot for fixing `exported` too!